File: matiasmasso/Mat.NET/DT2/DTOCategoriaDeNoticia.vb
Public Class DTOCategoriaDeNoticia
    Inherits DTOBaseGuid
    Property Nom As String
    Property Excerpt As String
    Property FchLastEdited As DateTime

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub

    Public Shadows Function ToString() As String
        Return _Nom
    End Function

End Class

File: matiasmasso/Mat.NET/DTO/DTOCategoriaDeNoticia.cs
using MatHelperStd;
using System;

namespace DTO
{
    public class DTOCategoriaDeNoticia : DTOBaseGuid
    {
        public string Nom { get; set; }
        public string Excerpt { get; set; }
        public DateTime FchLastEdited { get; set; }

        public DTOCategoriaDeNoticia() : base()
        {
        }

        public DTOCategoriaDeNoticia(Guid oGuid) : base(oGuid)
        {
        }

        public new string ToString()
        {
            return Nom;
        }

        public string Url(DTOWebDomain domain)
        {
            if (domain == null) domain = DTOWebDomain.Default();
            string retval = domain.Url("noticias", TextHelper.RemoveDiacritics(this.Nom));
            return retval;
        }

    }
}
